.circleci/config.yml

```diff
@@ -312,3 +319,7 @@ workflows:
       - simulate_eth_021
       - simulate_eth_022
       - simulate_base_003
+      - just_simulate_permissionless_fp_upgrade
+
```

Comment:
NIT: Can remove the space here 
Thanks for the conflits merging :)

Reply:
fixed